cookbook/import_export/create-connector.rst

```diff
@@ -42,20 +49,43 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
 
 We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
-.. note::
+.. warning::
 
-   It is also possible to create multiple job configuration files by placing them in a folder ``Resources/config/batch_jobs``. The job configurations must be yaml files of any name.
+   Please note that in version < 1.6, the file was always named "batch_jobs.yml" and was automatically loaded. The file content was very strict and was less evolutive than the more standard way to declare services that we use now.
 
-   For instance, I can declare a job configuration in a file named ``custom_jobs.yml`` like so ``Resources/config/batch_jobs/custom_jobs.yml``.
+Configure our JobParameters
+---------------------------
 
-Translate Job and Step titles
------------------------------
+To be executed, a Job is launched with a JobParameters which contains runtime parameters.
 
-Create a file ``Resources/translations/messages.en.yml`` in our Bundle to translate title keys.
+In our example, let's assume that our job needs a file path to write data, this file path must be provided and related to a writable directory to execute the job.
 
-.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Resources/translations/messages.en.yml
+We have to define a couple of services implementing ``Akeneo\Component\Batch\Job\JobParameters\DefaultValuesProviderInterface`` and ``Akeneo\Component\Batch\Job\JobParameters\ConstraintCollectionProviderInterface``.
+
+The first service provides the default values used to create a JobParameters,
+
+.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Job/JobParameters/DefaultValuesProvider/DummyExport.php
+    :language: php
+    :linenos:
+
+The second service provides the contraints used to validate each parameter of a JobParameters,
+
+.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Job/JobParameters/ConstraintCollectionProvider/DummyExport.php
+    :language: php
+    :linenos:
+
+These services are configured using tags to be properly registered and both declare a supports method to be used for our job only.
```

Comment:
Are using tags (no need to add "to be properly registered") and implement `supports()` method so they can only be used for our job.